runtime-modules/content/src/lib.rs

```diff
@@ -101,8 +106,25 @@ pub trait Trait:
 
     /// The storage type used
     type DataObjectStorage: storage::DataObjectStorage<Self>;
+
+    /// Video migrated in each block during migration
+    type VideosMigratedEachBlock: Get<u64>;
```

Comment:
nit picking here, but a better name could be:
```suggestion
    type VideoMigrationsPerBlock: Get<u64>;
```